### PR TITLE
Added New CheatSheet Of HTML

### PR DESCRIPTION
**Pull request by pralinkhaira**
## What does this PR do?
I added a new CheatSheet `HTML 5 - The Mega CheatSheet` by `Jamie Spencer` and is a pdf file  (For **hacktober fest**).

### Description
- This CheatSheet is free on `Make A Website Hub` Website and they are also distributing it for free through their [**website download page**](https://makeawebsitehub.com/the-html-5-mega-cheat-sheet/).
- Hope it helps (My 1st contribution to this repo)
### Why is this valuable (or not)?
- This **Mega CheatSheet** contains 99.9% all the elements of HTML updated till date.
- This will help you incase of a useful keyboard shortcut to forget, a command you will just fail to remember, a newly introduced function that slips your mind or element you cease to think of.
### How do we know it's really free?
- You can check out [Make A Website Hub](https://makeawebsitehub.com/the-html-5-mega-cheat-sheet/) and as soon as you land, there is an option `DOWNLOAD A4 PDF OF THIS HTML 5 MEGA CHEAT SHEET HERE!`.
### For book lists, is it a book? For course lists, is it a course? etc.
Yes, CheatSheet, HTML 5 Mega CheatSheet .

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates. **-Done**
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by pralinkhaira** [comment]
Merge it please, I want to make furthermore 3 PR.

File: more/free-programming-cheatsheets.md [hunk 1]
```diff
@@ -193,6 +193,7 @@
 * [FLEX: A simple visual cheatsheet for flexbox](https://flexbox.malven.co) - Chris Malven (HTML)
 * [GRID: A simple visual cheatsheet for CSS Grid Layout](https://grid.malven.co) - Chris Malven (HTML)
 * [HTML & CSS Emmet Cheat Sheet](https://docs.emmet.io/cheat-sheet/) - Emmet Documentation (HTML, [PDF]( https://docs.emmet.io/cheatsheet-a5.pdf))
+* [HTML 5 - The Mega CheatSheet](https://makeawebsitehub.com/wp-content/uploads/2015/06/HTML5-Mega-Cheat-Sheet-A4-Print-ready.pdf) - Jamie Spencer, [Make A Website Hub](https://makeawebsitehub.com/the-html-5-mega-cheat-sheet/) (PDF)
```
Comment (eshellman) [inline more/free-programming-cheatsheets.md:196]:
```suggestion
* [HTML 5 - The Mega CheatSheet](https://makeawebsitehub.com/wp-content/uploads/2015/06/HTML5-Mega-Cheat-Sheet-A4-Print-ready.pdf) - Jamie Spencer (PDF)
```

File: more/free-programming-cheatsheets.md [hunk 2]
```diff
@@ -193,6 +193,7 @@
 * [FLEX: A simple visual cheatsheet for flexbox](https://flexbox.malven.co) - Chris Malven (HTML)
 * [GRID: A simple visual cheatsheet for CSS Grid Layout](https://grid.malven.co) - Chris Malven (HTML)
 * [HTML & CSS Emmet Cheat Sheet](https://docs.emmet.io/cheat-sheet/) - Emmet Documentation (HTML, [PDF]( https://docs.emmet.io/cheatsheet-a5.pdf))
+* [HTML 5 - The Mega CheatSheet](https://makeawebsitehub.com/wp-content/uploads/2015/06/HTML5-Mega-Cheat-Sheet-A4-Print-ready.pdf) - Jamie Spencer, (PDF)
```
Comment (eshellman) [inline more/free-programming-cheatsheets.md:196]:
```suggestion
* [HTML 5 - The Mega CheatSheet](https://makeawebsitehub.com/wp-content/uploads/2015/06/HTML5-Mega-Cheat-Sheet-A4-Print-ready.pdf) - Jamie Spencer (PDF)
```

**Comment by pralinkhaira** [comment]
Done the changes!!! Please give it a review!

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**